### DISCUSSION TITLE
Button Style Oval Unwanted Outline

### DISCUSSION THREAD

**Discussion by arthurmelton** [fltk-rs/fltk-rs#952] [Q&A] [answered]
So I cant tell if I am just dumb but any way I can stop having this box around my buttons 
![image](https://user-images.githubusercontent.com/29708070/138533124-0d1e0391-265d-4e07-be04-3b19ffa0c632.png)

**Comment by MoAlyousef** [comment, marked as answer]
Yes. `btn.clear_visible_focus();`
